### PR TITLE
feat: uniform favicon, remove ARCHIVE_NODE requirement, and add Steps component 

### LINKED ISSUE #27: feature request: icon size and lettering size on Taiko docs tab
Icons and lettering are a different size on page tab when compared to other page tabs https://docs.taiko.xyz/start-here/getting-started

image

### PR DESCRIPTION
**Pull request by brachsterX**
Resolves #27 and #110 

Updates Run a Holesky node as well.

### CODE REVIEW

File: src/content/docs/guides/run-a-taiko-node.mdx [hunk 1]
```diff
@@ -14,131 +15,181 @@ This guide will help you start up a Taiko RPC node using simple-taiko-node.
...
-    ```sh
-    git clone https://github.com/taikoxyz/simple-taiko-node.git
-    cd simple-taiko-node && git config core.autocrlf false
-    ```
-  </TabItem>
-</Tabs>
+<Steps>
 
-### 2. Copy the sample .env files
+1. Install `stn`:
```
Comment (RogerLamTd) [inline src/content/docs/guides/run-a-taiko-node.mdx:23]: @brachsterX we've temporarily removed this part of the guide as stn hasn't yet been updated to support hekla

Comment (d1onys1us) [reply src/content/docs/guides/run-a-taiko-node.mdx:23]: yeah ill make sure to add it back once i add hekla support for stn. for now lets just use simple-taiko-node

Comment (brachsterX) [reply src/content/docs/guides/run-a-taiko-node.mdx]: I've updated the code 👍

Comment (RogerLamTd) [reply src/content/docs/guides/run-a-taiko-node.mdx]: @brachsterX also in enable-a-proposer guide :)

Comment (brachsterX) [reply src/content/docs/guides/run-a-taiko-node.mdx]: I can't seem to change on my end or I may be doing something wrong @RogerLamTd D:

Comment (RogerLamTd) [reply src/content/docs/guides/run-a-taiko-node.mdx]: > I can't seem to change on my end or I may be doing something wrong @RogerLamTd D:

Are you unable to just delete it?

Comment (brachsterX) [reply src/content/docs/guides/run-a-taiko-node.mdx]: enable-a-proposer has may have been resolved but the conflict now is the  src/content/docs/guides/run-a-holesky-node.md file which is already deleted

Comment (RogerLamTd) [reply src/content/docs/guides/run-a-taiko-node.mdx]: > enable-a-proposer may have been resolved but the conflict now is the src/content/docs/guides/run-a-holesky-node.md file which is already deleted

I see, could you merge main into your branch and resolve locally?

Comment (brachsterX) [reply src/content/docs/guides/run-a-taiko-node.mdx]: Thanks for guiding me on this @RogerLamTd , however it is still giving the same conflict.

Comment (RogerLamTd) [reply src/content/docs/guides/run-a-taiko-node.mdx]: @brachsterX it might be better to start fresh and cherry pick your changes to a new branch

Comment (brachsterX) [reply src/content/docs/guides/run-a-taiko-node.mdx]: created a new one on #113 . I'll close this PR.

**Pull request closed without merging**